lib/queries/index.js

```diff
@@ -0,0 +1,3 @@
+const { loadQuery } = require('../utils')
```

Comment:
GCI Leaders now using Babel, isn't it? Use `import` instead?

Reply:
Only frontend code (`/static`) uses Babel at the moment, so this isn't yet possible.